### DISCUSSION TITLE
How does one change the name appearing before "add project files" under a specific repository?

### DISCUSSION THREAD

**Discussion by ExcessPhase** [community/community#140322] [New to GitHub]
![image](https://github.com/user-attachments/assets/886fed0a-a6b1-4651-8196-b68d4b89ba7c)

In this case the name is "Frank Puck" which I used as a moniker some time ago.

**Comment by IsNotRicardo** [comment]
In the terminal of your local repository you want to run:

`git config user.name "New Name"`

In case you want to change the name for all your repositories you can run:

`git config --global user.name "New Name"`

Note that this will not change the name on previous commits, only on future ones. It is possible to amend previous commits, but it requires more work. You can read about it here: [How can I change the author name / email of a commit?](https://www.git-tower.com/learn/git/faq/change-author-name-email)
